ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ReportConfiguration.tsx

```diff
@@ -24,8 +24,7 @@ function ReportConfiguration(): ReactElement {
                         <Title headingLevel="h2">Report</Title>
                     </FlexItem>
                     <FlexItem>
-                        Optionally configure e-mail delivery destinations for manually triggered
-                        reports
+                        Optionally configure e-mail delivery destinations for reports
```

Comment:
Maria wanted to remove the word "manually" so thought I'd add it here instead of making a separate PR. (cc: @mansursyed)